tests/neg-macros/quote-type-variable-no-inference.check

```diff
@@ -3,7 +3,8 @@
   |                 ^
   |                 Ignored bound <: Double
   |
-  |                 Consider defining bounds explicitly `'{ type t <: Int & Double; ... }`
+  |                 This kind of pattern will be supported with SIP-53.
```

Comment:
Could the message mention the experimental import or link to the doc page on it if we have one?

Reply:
I noticed that this message can only be shown in an experimental context. We do not need this extra warning.

The only way to have this way to get code like this is using one of the two addition of SIP-53. Those are caught just before this check in https://github.com/lampepfl/dotty/pull/18199/files#diff-2eae364f82b9f4896588e75bfe0bdda905455d0d76136914f7a8909005048bf2R174-R177